diff -Naur a/drivers/thermal/Kconfig b/drivers/thermal/Kconfig
--- a/drivers/thermal/Kconfig	2021-07-07 22:05:13.128767172 +0800
+++ b/drivers/thermal/Kconfig	2021-07-08 06:35:34.610190826 +0800
@@ -510,12 +510,4 @@
 	  If you say yes here you get support for the FAN controlled
 	  by the Microcontroller found on the Khadas VIM boards.
 
-config AL_THERMAL
-	bool "Annapurna Labs thermal sensor driver"
-	depends on ARCH_ALPINE
-	depends on OF
-	help
-	  Enable this to plug the Annapurna Labs thermal sensor driver into the
-	  Linux thermal framework
-
 endif
diff -Naur a/drivers/thermal/Makefile b/drivers/thermal/Makefile
--- a/drivers/thermal/Makefile	2021-07-07 22:05:13.128767172 +0800
+++ b/drivers/thermal/Makefile	2021-07-08 06:36:19.417602590 +0800
@@ -62,4 +62,3 @@
 obj-$(CONFIG_AMLOGIC_THERMAL)     += amlogic_thermal.o
 obj-$(CONFIG_SPRD_THERMAL)	+= sprd_thermal.o
 obj-$(CONFIG_KHADAS_MCU_FAN_THERMAL)	+= khadas_mcu_fan.o
-obj-$(CONFIG_AL_THERMAL)	+= al/
